cmp/compare.go

```diff
@@ -66,21 +67,22 @@ import (
 //     Functions are only equal if they are both nil, otherwise they are unequal.
 //
 // Structs are equal if recursively calling Equal on all fields report equal.
-// If a struct contains unexported fields, Equal panics unless an Ignore option
-// (e.g., cmpopts.IgnoreUnexported) ignores that field or the Exporter option
-// explicitly permits comparing the unexported field.
+// If a struct contains unexported fields, Equal panics unless an [Ignore] option
+// (e.g., [github.com/google/go-cmp/cmp/cmpopts.IgnoreUnexported]) ignores that field
```

Comment:
This is extremely gross.

In my opinion, `go/doc` should also gather up related packages from the unit tests as well. That would be easiest and most concise way to fix reverse dependencies.

Reply:
Regardless, this is better than the status quo.

Alternatively, we can consider https://github.com/google/go-cmp/issues/136.